src/arguments/invite.ts

```diff
@@ -0,0 +1,21 @@
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { Argument } from '@sapphire/framework';
+
+export default class UserArgument extends Argument<string> {
+	private readonly kRegExp = /^(?:https?:\/\/)?(?:www.)?(?:discord\.gg\/|discordapp\.com\/invite\/)?(?<code>[\w\d-]{2,})$/i;
```

Comment:
You may want to move over to `discord-utilities` for this regex.

Reply:
Pretty sure this is in fact already there @kyranet